.github/pull_request_template.md

```diff
@@ -4,51 +4,46 @@ Thank you for contributing to TiKV!
 If you haven't already, please read TiKV's [CONTRIBUTING](https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md) document.
 
 If you're unsure about anything, just ask; somebody should be along to answer within a day or two.
+
+PR Title Format:
+1. pkg [, pkg2, pkg3]: what's changed
+2. *: what's changed
 -->
 
-###  What have you changed?
+### What problem does this PR solve?
 
-Please explain in detail what the changes are in this PR and why they are needed:
+Issue Number: close #xxx <!-- REMOVE this line if no issue to close -->
 
-- Summarize your change
-- How does this PR work? Include a brief introduction for any changed logic (optional)
-- Separately describe each logical change (optional)
-- Describe any limitations of the current code (optional)
+Problem Summary:
 
-Don't assume reviewers understand the original issue.
+### What is changed and how it works?
 
-###  What is the type of the changes?
-<!--
-Pick one of the following and delete the others:
--->
-- New feature (a change which adds functionality)
-- Improvement (a change which is an improvement to an existing feature)
-- Bugfix (a change which fixes an issue)
-- Engineering (engineering change which doesn't change any feature or fix any issue)
-- Misc (other changes)
+Proposal: [xxx](url) <!-- REMOVE this line if not applicable -->
+
+What's Changed:
```

Comment:
Usually these two share common content and hard to be separated.

Reply:
Do you mean "What's Changed" and "How it Works" should be combined into one section?

Reply:
Yes. According to my experience, they are hardly separated.